charts/kube-prometheus-stack/README.md

```diff
@@ -84,6 +84,10 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 12.x to 13.x
+
+The chart now requires helm v3.4.0 and later.
```

Comment:
```suggestion
The chart now requires Helm v3.4.0 and later.
```

Could you please elaborate why?

Reply:
Good point. I just reported the changes from upstream grafana chart.
Let's see https://github.com/grafana/helm-charts/pull/69#issuecomment-739954913

Reply:
Missed that one, thanks!

Reply:
Hm... I guess thats a misake.
Regarding the changes of the grafana pr, it should be only a dependency to 3.x.

Reply:
Oh, that's nice !

Reply:
@monotek I think you are right. The only thing I am not sure about is the `lookup` function used in https://github.com/grafana/helm-charts/pull/62/files#diff-84c93e8e7e63bcd29c4541c62905a4298731dd365dcafd95ce67be22d4e0438bR109, but that is not merged yet.

Reply:
Lookup function was introduced with helm 3.1.0, if the set milestone in the pr below is right:

https://github.com/helm/helm/pull/6752

Reply:
I opened https://github.com/grafana/helm-charts/pull/126